core/lib/presentation/views/list/sliver_grid_delegate_fixed_height.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/rendering.dart';
+
+class SliverGridDelegateFixedHeight
```

Comment:
this is for what?

Reply:
Custom widget builder for tile of gridview